src/functions/ducklake_options.cpp

```diff
@@ -22,6 +22,7 @@ static constexpr const ducklake_option_array DUCKLAKE_OPTIONS = {
      {"parquet_compression_level", "Compression level for Parquet files"},
      {"parquet_row_group_size", "Number of rows per row group in Parquet files"},
      {"parquet_row_group_size_bytes", "Number of bytes per row group in Parquet files"},
+     {"use_hive_partitioning", "If data should be partitioned in the hive format"},
```

Comment:
In DuckDB this is called `hive_file_pattern` - maybe it makes sense to keep that convention?

Reply:
Also maybe "`if partitioned data should be written in a hive-like folder structure`" is maybe more clear. The partitioning strategy used is unchanged after all  - the files will be partitioned in the same manner regardless of this setting.